### ISSUE TITLE
How to run evaluation only for TF 2.X Resnet50 model

### ISSUE THREAD

**Issue by ashiqimranintel** [tensorflow/models#9217]
I am using latest TF and TF model. Could you guide me through about how to run evaluation only with a Resnet50 checkpoint?

**Comment by saikumarchalla** [comment]
@ashiqimranintel  Could you please confirm if it is related to #9223. If so, Please close this issue and we can track the issue [here](https://github.com/tensorflow/models/issues/9223).Thanks!

**Comment by ashiqimranintel** [comment]
It's not same issue.

**Comment by jvishnuvardhan** [comment]
@ashiqimranintel Can you please share more details about your issue? Any standalone code would resolve the issue faster. Thanks!

**Comment by ashiqimranintel** [comment]
Hi @jvishnuvardhan, I was wondering how can I run evaluation only  with a checkpoint to see test accuracy. I don't want to train my model. Just want to skip training and run evaluation.

**Comment by jvishnuvardhan** [comment]
@ashiqimranintel Did you try loading checkpoint using `tf.keras.models.load_model` and evaluate the model? Thanks!
